### PR TITLE
fix(CardLink): remove specific styles

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#3473 

### Description

remove specific styles in web components

### Changelog

**Removed**

- web component styles

### CODE REVIEW

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @ariellalgilmore!

Update: Great if you can take a look at Percy diff for React, just in case it may be a true positive. Thanks!

File: packages/styles/scss/components/card/index.scss [hunk 1]
```diff
@@ -44,6 +44,7 @@
       @include carbon--type-style('body-short-02');
 
       margin-bottom: $carbon--spacing-07;
+      flex: 1;
```
Comment (kennylam) [inline packages/styles/scss/components/card/index.scss:47]: @ariellalgilmore Looks like this is causing the issue reported by @jeffchew

Comment (ariellalgilmore) [reply packages/styles/scss/components/card/index.scss]: @kennylam @jeffchew fixed!

**Comment by asudoh** [comment]
@jeffchew Seems that the visual regression has been fixed. Does this look good to go? Thanks!

**Review by jeffchew — APPROVED** [review]
LGTM, thank you for the fixes!

**Pull request merged**